proto/v2/observation.proto

```diff
@@ -57,9 +58,24 @@ message FacetFilter {
 
 // Generic observation request
 message ObservationRequest {
-  // A list of entity DCIDs
+  // A list of statistical variable DCIDs, or a formula expression.
+  //
+  // Supported operators for the formula expression: +, -, *, /, along with parentheses.
```

Comment:
nit: maybe move the examples to line 28 where formula is defined?

Reply:
done